### PR TITLE
RAXAJSS-261 schema for raxa alert created, .hbm.xml file and appropriate pojo created

### PR DESCRIPTION
**Pull request by TarangMahajan**
Created the schema (using liquibase.xml) for raxacore_raxaalert_list table.
Created RaxaCoreRaxaAlert.hbm.xml in the api/src/main/resources of the raxacore module and its appropriate RaxaAlert POJO.

### CODE REVIEW

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -66,4 +66,80 @@
...
+            </column>
+            <column defaultValueBoolean="false" name="voided" type="BOOLEAN">
+            <constraints nullable="false"/>
+            </column>
+            <column name="voided_by" type="int"/>
+            <column name="date_voided" type="DATETIME"/>
+            <column name="void_reason" type="varchar(255)"/>
+            <column name="changed_by" type="int"/>
+            <column name="date_changed" type="DATETIME"/>
+                <constraints unique="true" nullable="false"/>
```
Comment (johnstoecker) [inline api/src/main/resources/liquibase.xml:104]: date_changed does not need to be unique

File: api/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -66,4 +66,80 @@
...
+            <column name="name" type="varchar(255)"/>
+            <column name="description" type="varchar(2000)"/>
+            <column name="time" type="DATETIME">
+            <column defaultValueBoolean="false" name="read" type="BOOLEAN">
+            <column name="patient" type="varchar(255)"/>
+            <column name="provider_sent" type="varchar(255)"/>
+            <column name="provider_recipient" type="varchar(255)"/>
+            <column name="alert_type" type="varchar(255)"/>
+            <column name="default_task" type="varchar(255)"/>
+            <column name="uuid" type="char(38)">
```
Comment (johnstoecker) [inline api/src/main/resources/liquibase.xml:91]: We should have the 'read' boolean here as well

Comment (johnstoecker) [inline api/src/main/resources/liquibase.xml:91]: Actually, let's rename the variable 'seen' -- this variable is to check whether the recipient has read it or not....

File: api/src/main/resources/RaxaCoreRaxaAlert.hbm.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<hibernate-mapping package="org.raxa.module.raxacore">
+    
+    <class name="RaxaAlertList" table="raxacore_raxaalert_list">
+        <id name="raxaAlertId" column="raxa_alert_id">
+            <generator class="native"/>
+        </id>
+        <property name="name"                  type="java.lang.String"     column="name" length="255"/>
+        <property name="description"           type="java.lang.String"     column="description" length="2000"/>
+        <property name="time"                  type="java.util.Date"       column="time"/>
+        <property name="read"                  type="java.lang.Boolean"    column="read" length="1" not-null="true"/>
```
Comment (johnstoecker) [inline api/src/main/resources/RaxaCoreRaxaAlert.hbm.xml]: change this to 'seen'

File: api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    
+    private DateTime time;
+    
+    private Set<Obs> obs;
+    
+    private Set<Order> orders;
+    
+    private String providerSent;
+    
+    private String providerRecipient;
```
Comment (johnstoecker) [inline api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java]: We need the 'seen' variable here as well

File: api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+
+	private Integer raxaAlertId;
+
+    private String patient;
+    
+    private DateTime time;
+    
+    private Set<Obs> obs;
+    
+    private Set<Order> orders;
```
Comment (johnstoecker) [inline api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java]: Where is orders coming from? You are also missing alerttype, defaulttask, etc. Make sure to go through the required fields and only include what you want.

File: api/src/main/resources/liquibase.xml (hunk 1 above)
Comment (TarangMahajan) [inline api/src/main/resources/liquibase.xml:104]: edited

File: api/src/main/resources/liquibase.xml (hunk 2 above)
Comment (TarangMahajan) [inline api/src/main/resources/liquibase.xml:91]: read renamed to seen

File: api/src/main/resources/RaxaCoreRaxaAlert.hbm.xml (hunk 3 above)
Comment (TarangMahajan) [inline api/src/main/resources/RaxaCoreRaxaAlert.hbm.xml]: done

File: api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java (hunk 4 above)
Comment (TarangMahajan) [inline api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java]: k

File: api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java (hunk 5 above)
Comment (TarangMahajan) [inline api/src/main/java/org/raxa/module/raxacore/RaxaAlertList.java]: added the required and deleted the unnecessary.

**Comment by johnstoecker** [comment]
Please change all 'RaxaAlertList' to 'RaxaAlert'

**Comment by johnstoecker** [comment]
Closing this pull request, there are many errors, run through NetBeans and then issue a new pull request.

**Pull request closed without merging**